superset-frontend/src/components/Form/FormLabel.tsx

```diff
@@ -40,6 +40,7 @@ const RequiredLabel = styled.label`
   margin-bottom: ${({ theme }) => theme.gridUnit}px;
   &::after {
     display: inline-block;
+    margin-left: 4px;
```

Comment:
I think using grid unit here like you margin-bottom is preferred.

Reply:
Good catch! Fixed at [`this commit`](https://github.com/apache/superset/pull/14530/commits/16c52e443ce21981aa2312c3b773cfc6312566c9)